ocaml-lsp-server/test/e2e-new/code_actions.ml

```diff
@@ -1297,31 +1297,114 @@ let f (x : t) (d : bool) =
         "arguments": [
           "file:///foo.ml",
           {
-            "end": { "character": 0, "line": 3 },
-            "start": { "character": 0, "line": 3 }
+            "end": { "character": 2, "line": 4 },
+            "start": { "character": 2, "line": 4 }
           }
         ],
         "command": "ocamllsp/merlin-jump-to-target",
-        "title": "Jump to fun"
+        "title": "Match jump"
       },
-      "kind": "merlin-jump",
-      "title": "Jump to fun"
+      "kind": "merlin-jump-match",
+      "title": "Match jump"
     }
+
+       |}]
+;;
+
+let%expect_test "can jump to match-next-case target" =
+  let source =
+    {ocaml|
+type t = Foo of int | Bar of bool
+let square x = x * x
+let f (x : t) (d : bool) =
+  match x with
+  |Bar x -> x
+  |Foo _ -> d
+|ocaml}
+  in
+  let range =
+    let start = Position.create ~line:5 ~character:5 in
+    let end_ = Position.create ~line:5 ~character:5 in
+    Range.create ~start ~end_
+  in
+  print_code_actions source range ~filter:(find_action "merlin-jump-match-next-case");
+  [%expect
+    {|
+    Code actions:
     {
       "command": {
         "arguments": [
           "file:///foo.ml",
           {
-            "end": { "character": 2, "line": 4 },
-            "start": { "character": 2, "line": 4 }
+            "end": { "character": 3, "line": 6 },
+            "start": { "character": 3, "line": 6 }
           }
         ],
         "command": "ocamllsp/merlin-jump-to-target",
-        "title": "Jump to match"
+        "title": "Match-next-case jump"
```

Comment:
If the goal of the new name is to enable keyboard navigation in the code action you should try to have unique first letters like: 
`Match jump`
`Previous case jump`
`Next case jump`
`Let jump`
`Fun jump`

Reply:
will use a function to rename targets that begin with `match-`